website/pages/docs/agent/template/index.mdx

```diff
@@ -124,3 +124,39 @@ template {
 If you only want to use the Vault agent to render one or more templates and do
 not need to sink the acquired credentials, you can omit the `sink` stanza from
 the `auto_auth` stanza in the agent configuration.
+
+## Renewals and Updating Secrets
+
+The Vault Agent templating automatically renews and fetches secrets.  The behavior
```

Comment:
This behaviour differs somewhat from what Vault Agent cache does for renewals/checks.  I know you say "templating" here already, but could you call this out more explicitly to avoid confusion?

Reply:
Agreed, done!